ci/pipeline.ts

```diff
@@ -0,0 +1,24 @@
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import { SnykMonitorAddOn } from '../dist';
+
+const account = process.env.CDK_DEFAULT_ACCOUNT!;
```

Comment:
We're setting these same env vars to variables up in `bin/main.ts`. Can we extrapolate this logic into a settings file we include into both to follow the DRY principle?

Reply:
I added a commit that should explain what I mean. If we're handing the pipeline compiled assets I believe we'll be okay.

Reply:
Ahh, settings _object_ - now I see what you mean.